src/input/HidApiInputDriver.h

```diff
@@ -26,6 +26,7 @@
 
 #pragma once
 
+#include <tuple>
```

Comment:
Stray include?

Reply:
No, that was for `pair`, but it seems it should be actually `<utility>`.

Reply:
right, my brain just pattern-matched as one doesn't see tuple being used too often